### ISSUE TITLE
[Bug] Java error OverlappingFileLockException

### ISSUE THREAD

**Issue by gregdenay** [bio-raum/FooDMe2#49]
This is new:

```
ERROR ~ Error executing process > 'FOODME2:ILLUMINA_WORKFLOW:DADA2_WORKFLOW:HELPER_DADA_STATS (1)'

Caused by:
  java.nio.channels.OverlappingFileLockException
```

Any insights? Google is not very helpful

**Comment by marchoeppner** [comment]
What kind of file system are you running this one? Is this one a local drive or network-mounted (NFS, BeeGFS or similar?)

**Comment by gregdenay** [comment]
This is a VM on the server, I guess similar to a local disc?

I tried removing all the work, results and .nextflow folders, restart the server, clean mamba locks and caches and run the wrkflow in a new folder.

I get the same error on older versions of the pipeline too.

**Comment by marchoeppner** [comment]
Right, so then it is probably not the pipeline (seeing that it used to work previously) but something about the virtual machine and the attached storage (depending on what your setup looks like exactly). Happy to chat offline, but this might be beyond my area of expertise...

**Comment by gregdenay** [comment]
Fixed in [5fe9752](https://github.com/bio-raum/FooDMe2/pull/47/commits/5fe97528cea997a6e638802377c5eaa2955a4ece)

The naming of the conda env was the culprit. We have to ensure that the names are unique in the future :)
